sonar-plugin-api/src/main/java/org/sonar/api/server/rule/RulesDefinition.java

```diff
@@ -613,11 +613,35 @@ public String toString() {
    * Factory of {@link org.sonar.api.server.debt.DebtRemediationFunction}.
    */
   interface DebtRemediationFunctions {
+
+    /**
+     * Shortcut for {@code create(Type.LINEAR, coefficient, null)}.
+     * @param coefficient TODO. See {@link DebtRemediationFunction} for details about format.
+     * @see org.sonar.api.server.debt.DebtRemediationFunction.Type#LINEAR
+     */
     DebtRemediationFunction linear(String coefficient);
 
+    /**
+     * Shortcut for {@code create(Type.LINEAR_OFFSET, coefficient, offset)}.
+     * @param coefficient TODO. See {@link DebtRemediationFunction} for details about format.
+     * @param offset TODO. See {@link DebtRemediationFunction} for details about format.
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1135)